stdpopsim/cli.py

```diff
@@ -481,9 +489,7 @@ def add_simulate_species_parser(parser, species):
             help=(
                 "Specify a particular genetic map. By default, a chromosome-specific "
                 "uniform recombination rate is used. These default rates are listed in "
-                "the catalog: "
-                "<https://popsim-consortium.github.io/stdpopsim-docs/"
-                "stable/catalog.html> "
+                "the catalog: <https://stdpopsim.readthedocs.io/en/latest/catalog.html> "
```

Comment:
whoops - it looks like you're reverting some changes? We're using the github.io link for docs now.